src/pybind/mgr/dashboard/frontend/cypress/integration/orchestrator/workflow/02-create-cluster-add-host.e2e-spec.ts

```diff
@@ -0,0 +1,48 @@
+import { CreateClusterWelcomePageHelper } from 'cypress/integration/cluster/cluster-welcome-page.po';
+import { HostsPageHelper } from 'cypress/integration/cluster/hosts.po';
+
+describe('Create cluster add host page', () => {
+  const createCluster = new CreateClusterWelcomePageHelper();
+  const hosts = new HostsPageHelper();
+  const hostnames = ['ceph-node-00.cephlab.com', 'ceph-node-01.cephlab.com', 'ceph-node-02.cephlab.com'];
+  const addHost = (hostname: string, exist?: boolean) => {
+    hosts.navigateTo('add');
```

Comment:
This is not how hosts are added in the cluster creation workflow. Instead of navigating to the hosts add modal, it should press the Add Host button, and expects the modal to open up and enter these details in there.

Reply:
Yes I still need to adapt the tests with the add-host section. Will update it soon!